server/index.js

```diff
@@ -1,36 +1,18 @@
-dotenv.config();
 import express from 'express';
-import dotenv from 'dotenv';
-import cookieParser from 'cookie-parser';
-
-import { notFound, errorHandler } from './middleware/errorMiddleware.js';
-import connectDB from './database/connection.js';
-import userRoutes from './routes/userRoutes.js'
-import Product from './models/Product.js';
-
-const port = process.env.PORT || 4000;
-
+import dotenv from 'dotenv'
+import routes from './routes/routes.js'
+import db from './database/connection.js';
+dotenv.config();
 connectDB();
```

Comment:
Where is this function declared? There's no imports specified? Was this code tested?

Reply:
I don't how this is resolved? What is the `connectDB()` imported here?